homeassistant/components/homeassistant/translations/en.json

```diff
@@ -1,3 +1,17 @@
 {
-    "title": "Home Assistant"
+    "system_health": {
+        "info": {
+            "arch": "CPU Architecture",
+            "dev": "Development",
+            "docker": "Docker",
```

Comment:
```suggestion
            "chassis": "Chassis",
            "docker": "Docker",
            "docker_version": "Docker Version",
```

Reply:
Changing a keyname requires changing the return values in the system info helper.

Reply:
I did not change it ;)